docs/release-notes/next-lima/PT-168425449-lima-heights.md

```diff
@@ -0,0 +1 @@
+* Set Lima testnet hardfork height to 154400 (16th Oct 2019, 2pm CEST) and Lima mainnet to 161120 (30th Oct 2019, 2pm CEST)
```

Comment:
maybe split on two lines for better readability

Reply:
Will wait for the airdrop development and fix with final version